patterns/design-patterns/src/test/java/com/baeldung/currying/LetterUnitTest.java

```diff
@@ -0,0 +1,59 @@
+package com.baeldung.currying;
+
+import org.junit.Test;
+
+import java.time.LocalDate;
+
+import static com.baeldung.currying.Letter.LETTER_CREATOR;
+import static com.baeldung.currying.Letter.SIMPLE_CURRIED_LETTER_CREATOR;
+import static com.baeldung.currying.Letter.SIMPLE_LETTER_CREATOR;
+import static org.junit.Assert.assertEquals;
+
+public class LetterUnitTest {
+    private final static String BODY = "BODY";
+    private final static String SALUTATION = "SALUTATION";
+    private final static String RETURNING_ADDRESS = "RETURNING ADDRESS";
+    private final static String INSIDE_ADDRESS = "INSIDE ADDRESS";
+    private final static LocalDate DATE_OF_LETTER = LocalDate.of(2013, 3, 1);
+    private final static String CLOSING = "CLOSING";
```

Comment:
These should be `static final` not `final static`

Reply:
:heavy_check_mark: